webdriver-spec.html

```diff
@@ -4954,7 +4960,8 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
  <li><p>Let <var>start node</var> be the result
   of <a>trying</a> to <a>get a known connected element</a>
-  with argument <var>element id</var>.
+  with argument <var>element id</var>
+  from <var>url variables</var>.
```

Comment:
Throughout I would think of _url variables_ as  reference, not a variable. I think `with <a>url variable</a> <var>element id</var>` would flow better.

Possibly you can throw in an extra `<var>` if you feel like it: `<a><var>url variable</var</a> <var>element id</var>`.

Reply:
Yep! the `<var>url variable</var>` was a copy paste typo.

Done!